test/TestSpecification.cs

```diff
@@ -1,58 +1,48 @@
-using Xunit;
-using Xunit.Extensions;
+using System;
+using System.Threading.Tasks;
 
-[assembly: TestFramework("Xunit.Extensions.ObservationTestFramework", "Xunit.Bdd")]
-
-namespace Xunit.Bdd.Test
+namespace Xunit.Extensions.Test

```

Comment:
Why this namespace? The tests are specifically for our library, shouldn't they be in a corresponding namespace?

Reply:
mostly to not have to have a `using Xunit.Extensions` as the top

Reply:
That's, uh, a reason I guess... 🤷‍♂️